lucille-plugins/lucille-entity-extraction/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+      <groupId>com.kmwllc</groupId>
+      <artifactId>lucille-plugins</artifactId>
+      <version>0.5.7-SNAPSHOT</version>
+      <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>lucille-entity-extraction</artifactId>
+    <name>Lucille Plugins: Lucille Entity Extraction</name>
```

Comment:
We should declare     `<packaging>jar</packaging>`
This tells maven to build a jar for the module
It is the default packaging type, so things work when this declaration is left out, but our other plugin modules include it so we should do the same here to be consistent